### PR TITLE
Hyperledger Fabric: Initial integration

### PR DESCRIPTION
**Pull request by AdamKorcz**
This PR adds initial integration of Hyperledger Fabric. Hyperledger Fabric is an enterprise-grade permissioned distributed ledger framework for developing solutions and applications. Its modular and versatile design satisfies a broad range of industry use cases. It offers a unique approach to consensus that enables performance at scale while preserving privacy.

It is used by [a number of companies](https://www.hyperledger.org/learn/case-studies), including: 

- Bosch
- Walmart
- Honeywell
- German stock exchange group
- Sony Global Education
____________________________________________________________________

@sykesm @andrew-coleman @bestbeforetoday @lindluni @mastersingh24 Are you interested in integrating Hyperledger Fabric into OSS-fuzz for continuous fuzzing?

This PR adds the build files and the first fuzzer.

To complete the PR, a maintainers email address is needed for bug reports.

### CODE REVIEW

**Comment by mastersingh24** [comment]
@AdamKorcz Seems like a good idea to me.  Will let the others chime in as well.
You can use my email - gari.r.singh@gmail.com

**Comment by AdamKorcz** [comment]
Just checking in to see if there is anything to add from the Fabric maintainers, or can we proceed with the integration?

**Comment by DavidKorczynski** [comment]
It would be nice to get this one through - @mastersingh24 can we integrate this from Fabric's perspective? It would be nice to get the fuzzing going and then we can always improve with more fuzzers in the future.

**Comment by AdamKorcz** [comment]
@ale-linux @andrew-coleman @c0rwin @cendhu @denyeart @guoger @manish-sethi @yacovm Last year a null-pointer was discovered in Hyperledger Fabric by way of fuzzing: https://github.com/advisories/GHSA-vjj6-5m9f-wqjw. It would be great to complete Hyperledger Fabrics integration into OSS-Fuzz so that the code base can be fuzzed continuously. Can we add your email addresses for the contact list?

**Comment by ale-linux** [comment]
This sounds very interesting, I definitely support this. How can I help?

**Comment by AdamKorcz** [comment]
> This sounds very interesting, I definitely support this. How can I help?

We just need a maintainer email to get it started. Can we use yours? More can be added at any time later.

**Comment by ale-linux** [comment]
sure - go ahead! Thanks!

File: projects/fabric/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eu
+# Copyright 2021 Google LLC
```
Comment (jonathanmetzman) [inline projects/fabric/build.sh]: Wrong year

File: projects/fabric/persistence_fuzzer.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 Google LLC
```
Comment (jonathanmetzman) [inline projects/fabric/persistence_fuzzer.go]: Wrong year

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Review by DavidKorczynski — APPROVED** [review]

**Pull request merged**